### PR TITLE
add customer contact

### PR DESCRIPTION
**Pull request by ManishNarayan**
Add Customer Contact relation to Invoice Object. 
When you include customer for an invoice query -> platform also adds contact object to the response. to access this add customer_primary_contact relation.

### CODE REVIEW

File: app/platform_api/schema/invoice.rb [hunk 1]
```diff
@@ -280,6 +280,7 @@ def self.id_ref
   belongs_to :connection, {:class_name=>"Lockstep::Connection", :primary_key=>:company_id, :foreign_key=>"customer_id"}
   belongs_to :created_user, {:class_name=>"Lockstep::User", :primary_key=>:user_id, :foreign_key=>"created_user_id"}
   belongs_to :modified_user, {:class_name=>"Lockstep::User", :primary_key=>:user_id, :foreign_key=>"modified_user_id"}
+  belongs_to :customer_primary_contact, {:class_name=>"Lockstep::Contact", :included=>true}
```
Comment (shettytejas) [inline app/platform_api/schema/invoice.rb:283]: Won't the Lockstep::Invoice model be updated too?

File: app/platform_api/schema/invoice.rb [hunk 2]
```diff
@@ -5,17 +5,17 @@ def self.id_ref
   nil
 end
 
-  # The GroupKey uniquely identifies a single Lockstep Platform account.  All records for this
-  # account will share the same GroupKey value.  GroupKey values cannot be changed once created.
-  #             
+  # The GroupKey uniquely identifies a single Lockstep Platform account.  All records for this
```
Comment (shettytejas) [inline app/platform_api/schema/invoice.rb:8]: If the swagger.json was updated, can you keep the updated version in this commit. It's fine if the other models are updated too.

**Review by shettytejas — COMMENTED** [review]
Minor comments :)

File: app/platform_api/schema/invoice.rb (hunk 2 above)
Comment (ManishNarayan) [reply app/platform_api/schema/invoice.rb:8]: I did not make any changes. These must be just spaces. I dont see any changes in terms of words if I read

**Pull request merged**